code/modules/spells/spell_types/conjure.dm

```diff
@@ -31,7 +31,12 @@
 		if(ispath(summoned_object_type, /turf))
 			var/turf/O = spawn_place
 			var/N = summoned_object_type
-			O.ChangeTurf(N, flags = CHANGETURF_INHERIT_AIR)
+			if(istype(O, /turf/open) && ispath(N, /turf/closed))
+				new N(O)
+			if(istype(O, /turf/open/floor/plating))
```

Comment:
Is this not redundant with the previous?

Reply:
Well, not too sure since I'd need to look at the whole code of that thing again. But the reason I did it that way was because first one checks if the object being placed is a wall and if the spot that it is being placed on is a floor, if it is, the wall is placed on top of the floor without removing it. It doesn't actually keep the floor itself, but it does prevent plating from being removed when a wall is placed. The second was written to check if the object being placed on is specifically plating, and it doesn't check what is being placed, I mainly did it so the floors can still be replaced, instead of just floor layers stacking up.